### PR TITLE
Add Images UG

### PR DESCRIPTION
**Pull request by adeearyaa**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -261,8 +261,11 @@ Format: `find c/COMPANY | r/ROLE | s/STAGE`
 
 
 Examples:
-* `find r/Engineer` returns `Software Engineer` and `Backend Engineer`.
-* `find s/Interview` returns `Technical Interview` and `Behavioural Interview`.
+*`find r/Engineer` returns `Software Engineer` and `Backend Engineer`:
+  |       Before        |       After      |
+  |:-------------------------:|:-------------------------:|
+  |![before layout](images/BeforeFind.png) |![after layout](images/AfterFind.png) |
+
```
Comment (jacobkwan) [inline docs/UserGuide.md:268]: Screenshot 2022-11-06 at 2 15 52 PM

formatting error here, can be fixed with:
```suggestion
* `find r/Engineer` returns `Software Engineer` and `Backend Engineer`:

  |       Before        |       After      |
  |-------------------------|-------------------------|
  |![before layout](images/BeforeFind.png) |![after layout](images/AfterFind.png) |
  
```
which will give u this, but the images aren't really visibile imo
Screenshot 2022-11-06 at 2 18 58 PM

**Review by jacobkwan — COMMENTED** [review]
the window size of the images is too big resulting in the text being hard to read, maybe you can reduce the window width to the minimum required for wide layout?
also, whats the extra two .pngs for? i only see two being referenced in the UG

**Review by rylzxc — APPROVED** [review]
Thank u for not including the file header!!

**Review by jacobkwan — APPROVED** [review]
LGTM

**Pull request merged**